docs/custom_domains.rst

```diff
@@ -99,6 +99,19 @@ You can also host your documentation from your own domain.
 
          .. _Amazon CAA guide: https://docs.aws.amazon.com/acm/latest/userguide/setup-caa.html
 
+Strict Transport Security
++++++++++++++++++++++++++
+
+By default, we do not return a `Strict Transport Security header`_ (HSTS) for user custom domains.
+This is a conscious decision as it can be misconfigured in a not easily reversible way.
+For both |org_brand| and |com_brand|, HSTS for custom domains can be set upon request.
```

Comment:
Maybe is worth to mention how to make that request, with the link to the email?

Reply:
We could mention contacting support, but I think we don't need to include a link here, since it would be different for each site?